.github/actions/sil-kit-ci/execute/action.yml

```diff
@@ -0,0 +1,60 @@
+name: sil-kil-ci/execute
+
+description: >
+  Execute a command with arguments.
+  On Linux runners, the command is executed in a docker container using the `linux-docker-image`.
+  On Windows runners, the command is executed on the runner itself.
+
+inputs:
+
+  preset-name:
+    description: The preset name from CMakePresets.json.
+    required: true
+
+  linux-docker-image:
+    description: Docker image used under Linux
+    required: true
+
+  command:
+    description: Command to execute
+    required: true
+
+  args:
+    description: Arguments passed to the command
+    required: true
+
+runs:
+  using: composite
+  steps:
+
+    - name: compute directories
+      uses: ./.github/actions/sil-kit-ci/dirs
+      id: dirs
+      with:
+        preset-name: ${{ inputs.preset-name }}
+
+    - name: (linux) execute command
+      if: runner.os == 'Linux'
+      shell: bash
+      run: |
+        docker run \
+          -v "${{ steps.dirs.outputs.cmake-source }}:/s" \
+          -v "${{ steps.dirs.outputs.cmake-build }}:/b" \
+          -v "${{ steps.dirs.outputs.cmake-package }}:/p" \
+          -e "SIL_KIT_CI_CMAKE_BINARY_DIR=/b" \
+          -e "SIL_KIT_CI_CMAKE_INSTALL_DIR=/b/i" \
+          -e "SIL_KIT_CI_CMAKE_PACKAGE_DIR=/p" \
+          -w "/s" \
+          "${{ inputs.linux-docker-image }}" \
```

Comment:
Can't we just pass in the whole command line instead of splitting it into `command` and `args` ?

Reply:
I don't see why not.

Reply:
Done 👍